src/ddmd/mtype.d

```diff
@@ -2510,7 +2510,9 @@ extern (C++) abstract class Type : RootObject
         }
         else if (ident == Id.__xalignof)
         {
-            e = new IntegerExp(loc, alignsize(), Type.tsize_t);
+            const alignment = alignment();
```

Comment:
Using the same name for a variable as its initializer looks a little wonky. I have to look at it carefully to see if there's something going wrong here - shouldn't have to do that.